duplicates-in-array.js

```diff
@@ -0,0 +1,20 @@
+module.exports = function (array, n){
```

Comment:
Why do you need `n` as parameter? The problem description says: "Given an array of numbers having size N in which every number is between 1 and N". In my opinion, it doesn't state that you'll be given an array of numbers and a number representing the size or the number of elements.

Reply:
Yes it does not say, but I used n to verify if my given array has the number of items I want and to verify if each item is smaller than n.  I could have used only the array's length to verify if each number is smaller than the length.

Reply:
Given the description, you do not need to check if the array is a good one (that's others responsibility), you only have to check for duplicates. :smiley: